src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2713,6 +2737,52 @@ export class ResultsViewPageStore {
         },
     });
 
+    readonly structuralVariants = remoteData<StructuralVariant[]>({
+        await: () => [this.genes, this.samples, this.structuralVariantProfiles],
+        invoke: async () => {
+            const structuralVariantProfiles = this.structuralVariantProfiles
+                .result;
+
+            if (structuralVariantProfiles.length === 0) {
+                return [];
+            }
+
+            const studyIdToProfileMap: {
+                [studyId: string]: MolecularProfile;
+            } = _.keyBy(
+                structuralVariantProfiles,
+                (profile: MolecularProfile) => profile.studyId
+            );
+
+            const filters = this.samples.result.reduce(
+                (memo, sample: Sample) => {
+                    if (sample.studyId in studyIdToProfileMap) {
+                        memo.push({
+                            molecularProfileId:
+                                studyIdToProfileMap[sample.studyId]
+                                    .molecularProfileId,
+                            sampleId: sample.sampleId,
+                        });
+                    }
+                    return memo;
+                },
+                [] as any[]
+            );
+
+            const data = {
+                entrezGeneIds: _.map(
+                    this.genes.result,
+                    (gene: Gene) => gene.entrezGeneId
+                ),
+                sampleMolecularIdentifiers: filters,
+            } as StructuralVariantFilter;
+
+            return await client.fetchStructuralVariantsUsingPOST({
```

Comment:
just remove async/await and return client.fetch...    since we aren't actually awaiting anything